ui/src/plugins/es.json

```diff
@@ -0,0 +1,21 @@
+{
+  "welcomeMsg": "Five Fifths Voter",
+  "tagline": "5/5 es igual a un todo.",
+  "subtagline": "Toda una persona, toda una comunidad, toda una nación.",
+  "appHeaderWhyVote": "Por qué votar",
+  "appHeaderVoterJourney": "Viaje de votantes",
+  "appHeaderGetConnected": "Cómo conectarse",
+  "appHeaderVoterSupport": "Apoyo electoral",
+  "countdownHeader": "Elecciones legislativas intermedias",
+  "landingPageMain": "Nuestra Misión",
+  "landingPageDesc": "Con cinco quintos de votante, ayudamos a poner el poder en sus manos. Nuestra plataforma proporciona una experiencia personalizada con herramientas útiles para ayudar a los votantes marginados a superar los bloqueos de proceso de votación. Cinco quintos de votante tiene en cuenta varios viajes de votación, como los votantes por primera vez, los veteranos, las minorías, los discapacitados legales, los ancianos, o aquellos con condenas graves. Juntos, podemos cambiar la narrativa de las personas históricamente marginadas en nuestro proceso democrático.",
+  "landingPageEdu": "Educar.",
+  "landingPageQuote": "\"El conocimiento es poder.\"- Francis Bacon",
+  "landingPageEduDesc": "Cinco quintos de los votantes educarán a los votantes en derechos de voto con la guía de votación y la conexión a las herramientas actualizadas necesarias para completar el proceso de votación.",
```

Comment:
Product name should be extracted? possibly reword or have substitution character sequence in strings? 
cinco quintos de votante translation also lost Five Fifths Voter from brand/product into the statement.

Reply:
I am guessing that this rephtasing needs to be considered with the next wave of development effort.

Reply:
Yes, thank you @GeraldMit. We can use this - https://vue-i18n.intlify.dev/guide/essentials/syntax.html#named-interpolation

Is the product name translated at all? Taking hindi as example:

1. फाइव फिफ्थ वोटर. This reads as Five Fifth Voters, but the words do not mean anything in Hindi.
2. पांचवां मतदाता does not read Five Fifth Voters, but means five fifth voters when translated to English.
3. Leave as Five Fifth Voters. As in `Five Fifth Voter के साथ, हम शक्ति को वापस अपने हाथों में रखने में ..."

Reply:
I went with `#3` for now.
![image](https://user-images.githubusercontent.com/3187457/130703709-fb7ad9f5-b4be-4f8a-95f9-14ebe6a4b781.png)